src/RoboSaverVirtualModule.sol

```diff
@@ -19,38 +16,23 @@ import "@balancer-v2/interfaces/contracts/solidity-utils/misc/IERC4626.sol";
 import {KeeperCompatibleInterface} from "@chainlink/automation/interfaces/KeeperCompatibleInterface.sol";
 
 import {VirtualModule} from "./types/DataTypes.sol";
+import {Errors} from "./libraries/Errors.sol";
+
+import {VirtualModuleConstants} from "./abstracts/VirtualModuleConstants.sol";
```

Comment:
maybe call it `RoboSaverConstants.sol`?

Reply:
cb11a69d289d98ec49341c4affb8e317825450c6